### ISSUE TITLE
[pdfx] Can't upgrade pubspec because of the uuid package

### ISSUE THREAD

**Issue by vlazdra** [ScerIO/packages.flutter#428]
**Describe the bug**
As the title states, the dependency resolution fails because pdfx depends on 3.x.x and there is a new uuid major version available 4.0.0. [uuid CHANGELOG](https://pub.dev/packages/uuid/changelog)

**To Reproduce**
1. Add uuid
2. Add pdfx
3. Dependency resolution fails

**Expected behavior**
Dependency resolution succeeds

**Screenshots**
n/a

**Smartphone (please complete the following information):**
n/a

**Additional context**
n/a

**Comment by vlazdra** [comment]
Any news on this? @SergeShkurko

**Comment by SergeShkurko** [comment]
@vlazdra published v2.5.0

**Comment by emakar** [comment]
it's not yet published
https://pub.dev/packages/pdfx/versions

**Comment by ynnob** [comment]
@SergeShkurko Please look again. You didn't publish v2.5.0 to pub.dev

**Comment by Paroca72** [comment]
Same issue here

**Comment by emakar** [comment]
> @vlazdra published v2.5.0

@vlazdra could you please take a look at it?

**Comment by vlazdra** [comment]
Hey @emakar, I'm not the developer of the plugin. It seams that it's available on the master branch, which you can reference in your pubspec directly:
```yaml
  pdfx:
    git:
      url: https://github.com/ScerIO/packages.flutter
      ref: main
      path: packages/pdfx
```

Hey @SergeShkurko the new version is not available on Pub.dev: https://pub.dev/packages/pdfx
